### PR TITLE
OPENSVC-18758: Pharmacy Verification Status custom extension

### PR DESCRIPTION
**Pull request by maruthiallam**
Description
----

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.
Screen Shot 2021-04-07 at 1 57 24 PM
Screen Shot 2021-04-07 at 1 51 25 PM

### CODE REVIEW

**Review by fpivonka — APPROVED** [review]

File: content/millennium/r4/conformance/structure-definition.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Authorization is not required.
  [`formatted-text`]                                          | Link to the formatted text for a note. It is used in cases when the note is not a plain text.
  [`patient-friendly-display`]                                | An expression of the original string in terminology that patients should be able to understand.
  [`payment-collection-status`]                               | The status of the payment collection for the encounter.
+ [`pharmacy-verification-status`]                            | Represents whether a MedicationRequest has been verified by a pharmacist. Supported values are Does not need pharmacy verification, Needs pharmacy verification or Rejected by pharmacy.
```
Comment (joeratt) [inline content/millennium/r4/conformance/structure-definition.md:99]: The rest of the definitions don't seem to indicate the supported values. We might be able to just leave that as part of the Structure definition itself, as well as the MedicationRequest page.

Comment (maruthiallam) [reply content/millennium/r4/conformance/structure-definition.md]: Removed [commit](https://github.com/cerner/fhir.cerner.com/pull/609/commits/895786e1d26c542a51ad48e9bc22074506b9aac6)

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 2]
```diff
@@ -204,6 +210,12 @@ module Resources
             'display': 'Electronic'
           },
           'url': 'http://electronichealth.se/fhir/StructureDefinition/NLLRegistrationBasis'
+        },
+        {
+          'valueCodeableConcept': {
+            'text': 'Does not need pharmacy verification'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/pharmacy-verification-status'
```
Comment (joeratt) [inline lib/resources/example_json/r4_examples_medication_request.rb:218]: Is this the patient-access block? If so, this won't be returned, right?

Comment (maruthiallam) [reply lib/resources/example_json/r4_examples_medication_request.rb:218]: Good Catch @joeratt this should be removed from patient token workflow.  I will update it.

**Review by joeratt — APPROVED** [review]

**Review by fpivonka — APPROVED** [review]

**Comment by ebbp123** [comment]
This PR Can't be verified live as the changes here were reverted in https://github.com/cerner/fhir.cerner.com/pull/616

**Pull request merged**